releasenotes/notes/0.4/deprecate-classes-ef077e50db9718bc.yaml

```diff
@@ -79,7 +78,7 @@ upgrade:
 deprecations:
   - |
     :class:`~qiskit_ibm_runtime.IBMRuntimeService`, :class:`~qiskit_ibm_runtime.IBMEstimator`
-    and :class:`~qiskit_ibm_runtime.IBMSampler` classes have been deprecated and will be removed
-    in a future release. Use :class:`~qiskit_ibm_runtime.QiskitRuntimeService`,
+    and :class:`~qiskit_ibm_runtime.IBMSampler` classes have been deprecated and will be removed in Release 0.5.0.
+    Use :class:`~qiskit_ibm_runtime.QiskitRuntimeService`,
     :class:`~qiskit_ibm_runtime.Estimator` and :class:`~qiskit_ibm_runtime.Sampler` classes
     instead. See upgrade notes section for a detailed explanation with examples.
```

Comment:
We typically don't modify previous release notes, I would revert changes to this file and instead add a new release note using command `reno new remove-ibm-sampler` and explain why it is being removed.

Reply:
@dtquai Thanks could you please add a new release note using above instructions?